com.github.philip_scott.notes-up.json

```diff
@@ -0,0 +1,119 @@
+{
+	"app-id": "com.github.philip_scott.notes-up",
```

Comment:
replacing only the first dash with an underscore works, but shouldn't this actually be the other way around: https://github.com/flatpak/flatpak/issues/295#issuecomment-246260896 ?